README.md

```diff
@@ -1,26 +1,45 @@
 ## Description
-These rules refer to 
+These rules refer to http://developer.android.com/guide/developing/device.html and include many suggestions from the Archlinux and Github Community.
 
-http://developer.android.com/guide/developing/device.html
+If you're developing on Ubuntu Linux, you need to add a udev rules file that contains a USB configuration for each type of device you want to use for development. In the rules file, each device manufacturer is identified by a unique vendor ID, as specified by the ATTR{idVendor} property. For a list of vendor IDs, see USB Vendor IDs, below.
 
-and include many suggestions from the Archlinux and Github Community
+Use this format to add each vendor to the file(example vendor ID is for HTC):  
+`SUBSYSTEM=="usb", ATTR{idVendor}=="0bb4", MODE="0666", GROUP="plugdev"`
+
+The MODE assignment specifies read/write permissions, and GROUP defines which Unix group owns the device node.
+
+Note: The rule syntax may vary slightly depending on your environment. Consult the udev documentation for your system as needed. For an overview of rule syntax, see this guide to writing udev rules. please refer to 51-android.rules in [ubuntu directory](ubuntu)
```

Comment:
Again strike out  Ubuntu and replace with Linux / Unix /Mac  or  a simple  'non -windows'  env